build.gradle

```diff
@@ -1,11 +1,23 @@
+plugins {
+    id "org.jetbrains.kotlin.jvm" version "1.3.41"
```

Comment:
There should be a built-in shortcut for that, maybe `kotlin("jvm")`?

Reply:
No, this shortcut is only for gradle kts

Reply:
OK. I'm merging now, but next step: convert to kts.